### ISSUE TITLE
Bug: New tracker implemenation leads to error in overview

### ISSUE THREAD

**Issue by certrik** [ail-project/ail-framework#69]
Hi @Terrtia ,
just updated the AIL-framework to the last commits `fix: [Tracker] edit tracker ACL` (https://github.com/ail-project/ail-framework/commit/78903ec033ac2f2bc764972f3f20ad98c83b6768) , `chg: [Tracker] edit tracker` (https://github.com/ail-project/ail-framework/commit/4fa320741cbec40735a1efa1bb7904720cc5ad11) and so far everything very cool but using the overview in 
![grafik](https://user-images.githubusercontent.com/42176839/101889632-eacbdd80-3b9f-11eb-8f09-b3516cbb2337.png) is broken for regex and yara trackers. Clicking on those links leads to ... 
```
Internal Server Error

The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.
```

**Comment by Terrtia** [comment]
Hi @certrik !

Can you give me the output of the Flask Server  ? 
(``screen -r Flask_AIL``)

**Comment by certrik** [comment]
Of course certainly ...
```
[11/Dec/2020 15:41:30] "GET /trackers HTTP/1.1" 200 -
[2020-12-11 15:41:37,893] ERROR in app: Exception on /trackers/regex [GET]
Traceback (most recent call last):
  File "/data/AIL/ail-framework/AILENV/lib/python3.7/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/data/AIL/ail-framework/AILENV/lib/python3.7/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/data/AIL/ail-framework/AILENV/lib/python3.7/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/data/AIL/ail-framework/AILENV/lib/python3.7/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/data/AIL/ail-framework/AILENV/lib/python3.7/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/data/AIL/ail-framework/AILENV/lib/python3.7/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/data/AIL/ail-framework/AILENV/lib/python3.7/site-packages/flask_login/utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "./modules/Role_Manager.py", line 98, in decorated_view
    return func(*args, **kwargs)
  File "/data/AIL/ail-framework/var/www/modules/hunter/Flask_hunter.py", line 78, in tracked_menu_regex
    return render_template("trackersManagement.html", user_term=user_term, global_term=global_term, bootstrap_label=bootstrap_label, filter_type=filter_type)
  File "/data/AIL/ail-framework/AILENV/lib/python3.7/site-packages/flask/templating.py", line 140, in render_template
    ctx.app,
  File "/data/AIL/ail-framework/AILENV/lib/python3.7/site-packages/flask/templating.py", line 120, in _render
    rv = template.render(context)
  File "/data/AIL/ail-framework/AILENV/lib/python3.7/site-packages/jinja2/environment.py", line 1090, in render
    self.environment.handle_exception()
  File "/data/AIL/ail-framework/AILENV/lib/python3.7/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "/data/AIL/ail-framework/AILENV/lib/python3.7/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "/data/AIL/ail-framework/var/www/modules/hunter/templates/trackersManagement.html", line 70, in top-level template code
    {% if dict_uuid['term']|length > 256 %}
TypeError: object of type 'NoneType' has no len()
[11/Dec/2020 15:41:38] "GET /trackers/regex HTTP/1.1" 500 -
[11/Dec/2020 15:41:38] "GET /favicon.ico HTTP/1.1" 404 -

```

**Comment by Terrtia** [comment]
Thank you !
I pushed a quick fix 28f6963ff402124ef40da0e527241b7b37219c75. 
It seem like some trackers are Null. Can you check on the UI if one of your tracker is empty ?

**Comment by certrik** [comment]
You are welcome and thank you for your great AIL-framework.
I just checked and at least there are no global trackers neither my personal trackers that are Null.

**Comment by certrik** [comment]
> 
> 
> Thank you !
> I pushed a quick fix [28f6963](https://github.com/ail-project/ail-framework/commit/28f6963ff402124ef40da0e527241b7b37219c75).
> It seem like some trackers are Null. Can you check on the UI if one of your tracker is empty ?

This commit fixed it for me.
